jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/content/AsyncContent.java

```diff
@@ -68,6 +81,7 @@ public void write(boolean last, ByteBuffer byteBuffer, Callback callback)
      * write is complete.</p>
      * <p>The callback completes:</p>
      * <ul>
+     * <li>immediately with a failure when this instance is closed or in error</li>
```

Comment:
Please specify you mean "already" in error.

Reply:
Done.